tools/ci/gha-caches.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -ex
+
+echo "NJOY_HASH=$(git ls-remote https://github.com/njoy/NJOY2016.git | cut -f 1 |\
+    head -c 15)" >> $GITHUB_ENV
```

Comment:
You can use the form `git ls-remote <url> <ref>` where `<ref>` is `HEAD` to get the first line (i.e., `git ls-remote https://github.com/njoy/NJOY2016.git HEAD | head -c 15`)